look and feel logo upload error

console logs an error, but everything appears to work as expected

```
bundle.js:1 Error: [$compile:nonassign] http://errors.angularjs.org/1.6.5/$compile/nonassign?p0=undefined&p1=fileModel&p2=dropzone
    at bundle.js:1
    at F (bundle.js:1)
    at q (bundle.js:1)
    at e (bundle.js:1)
    at m.$digest (bundle.js:1)
    at m.$apply (bundle.js:1)
    at Object.$$debounceViewValueCommit (bundle.js:1)
    at Object.$setViewValue (bundle.js:1)
    at m (bundle.js:1)
    at bundle.js:1
```

--- a/src/main/webapp/app/views/admin/settings/application/lookAndFeel.html
+++ b/src/main/webapp/app/views/admin/settings/application/lookAndFeel.html
@@ -298,7 +298,7 @@
 
 
 				<div class="col-sm-4">
-					<dropzone id="rightLogoDrop" text="uploadDataText" patterns="uploadDataPattern" drop-method="previewLogo(files)"></dropzone>
+					<dropzone file-model="modalData.newLogo.file" id="rightLogoDrop" text="uploadDataText" patterns="uploadDataPattern" drop-method="previewLogo(files)"></dropzone>
 				</div>
 
 			</div>

